### PR TITLE
Update suggested mozconfig in docs to use CC/CXX from ~/.mozbuild and lld

### PR DESCRIPTION
**Pull request by snorp**

### CODE REVIEW

File: tutorials/geckoview-quick-start.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ ac_add_options --with-java-bin-path="/Library/Java/Home/bin"
 ac_add_options --with-android-sdk="$HOME/.mozbuild/<your-sdk>"
 ac_add_options --with-android-ndk="$HOME/.mozbuild/android-ndk-r17b"
 
+ac_add_options --enable-linker=lld
```
Comment (agi90) [inline tutorials/geckoview-quick-start.md:57]: I wonder if we need this section given that boostrap already tells you to add these lines:

```
>>>
# Build GeckoView/Firefox for Android:
ac_add_options --enable-application=mobile/android

# Targeting the following architecture.
# For regular phones, no --target is needed.
# For x86 emulators (and x86 devices, which are uncommon):
# ac_add_options --target=i686
# For newer phones.
# ac_add_options --target=aarch64
# For x86_64 emulators (and x86_64 devices, which are even less common):
# ac_add_options --target=x86_64

# With the following java:
ac_add_options --with-java-bin-path="/Library/Java/Home/bin"

# With the following Android NDK:
ac_add_options --with-android-ndk="/Users/asferro/.mozbuild/android-ndk-r17b"

# With the following compiler toolchain:
CC="/Users/asferro/.mozbuild/clang/bin/clang"
CXX="/Users/asferro/.mozbuild/clang/bin/clang++"
<<<
```

**Review by agi90 — APPROVED** [review]

File: tutorials/geckoview-quick-start.md (hunk 1 above)
Comment (snorp) [reply tutorials/geckoview-quick-start.md:57]: I don't think it hurts -- I think people are mostly following this step-by-step guide and not necessarily looking too closely at the output of the commands.

Comment (agi90) [reply tutorials/geckoview-quick-start.md:57]: yeah I agree. I'm just worried that this output will change and the documentation will be stale again.

Comment (fluffyemily) [reply tutorials/geckoview-quick-start.md:57]: In hindsight, I agree with @agi90 here. Alvina's experience has demonstrated that the output does change and our docs here will cause issues to people trying to build for the first time. 

If no one objects, I shall close this PR and raise a new one removing this section.

**Pull request closed without merging**